apps/web/src/store/publication.ts

```diff
@@ -20,5 +24,14 @@ export const usePublicationStore = create<PublicationState>((set) => ({
   publicationContent: '',
   setPublicationContent: (publicationContent) => set(() => ({ publicationContent })),
   audioPublication: { title: '', author: '', cover: '', coverMimeType: 'image/jpeg' },
-  setAudioPublication: (audioPublication) => set(() => ({ audioPublication }))
+  setAudioPublication: (audioPublication) => set(() => ({ audioPublication })),
+  attachments: [],
+  setAttachments: (attachments) => set(() => ({ attachments })),
+  addAttachments: (newAttachments) =>
+    set((state) => {
+      if (newAttachments.length === 0) {
+        return { attachments: [...state.attachments] };
```

Comment:
What is the use of this condition? If there are no attachments then it should ideally return the same state only right?